src/sdk/uservarmanager.cpp

```diff
@@ -260,19 +199,24 @@ void UserVariableManager::Arrogate()
                      "%s\n\n"
                      "Please define them."), peList.wx_str());
 
-    PlaceWindow(&d);
+    m_ui->DisplayInfoWindow(_("Global Compiler Variables"), msg);
+    m_ui->OpenEditWindow(m_Preempted);
     m_Preempted.Clear();
-    InfoWindow::Display(_("Global Compiler Variables"), msg , 8000 + 800*m_Preempted.GetCount(), 100);
-
-    d.ShowModal();
 }
 
 UserVariableManager::UserVariableManager()
 {
     m_CfgMan = Manager::Get()->GetConfigManager(_T("gcv"));
+    m_ui =  new UserVarManagerDefaultUI(); //new UserVarManagerGUI(); //new UserVarManagerDefaultUI();
```

Comment:
No commented code please. No extra space after the space... More focus.

Reply:
fixed in bf2875c